src/main/java/org/kiwiproject/dropwizard/poller/health/ClientPollerHealthChecks.java

```diff
@@ -96,4 +104,75 @@ public static Result unhealthy(ClientPollerStatistics statistics, HealthStatus s
                 .withDetail(FAILURE_DETAILS_KEY, statistics.recentFailureDetails().collect(toUnmodifiableList()))
                 .build();
     }
+
+    /**
+     * Convenience method that registers the following health checks for the given poller and Dropwizard environment:
+     * {@link ClientPollerTimeBasedHealthCheck}, {@link ClientPollerLatencyBasedHealthCheck}, and
+     * {@link ClientPollerMissedPollHealthCheck}
+     *
+     * @param poller      The poller being monitored
+     * @param environment The dropwizard environment
+     */
+    public static void registerPollerHealthChecks(ClientPoller poller, Environment environment) {
```

Comment:
Is there any reason we might want to return a list containing the registered health checks?

Reply:
Reworked this